cylc/flow/cfgspec/workflow.py

```diff
@@ -1175,6 +1175,14 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                     "[platforms][<platform name>]submission retry delays"
                 )
             )
+            Conf(
+                'run mode', VDR.V_STRING,
+                options=['workflow', 'simulation', 'dummy', 'live', 'skip'],
+                default='workflow',
+                desc='''
+                    Set an individual task's run mode. By default this
+                    will pick up the run mode from the workflow.
```

Comment:
This ought to either give more explanation of what task run mode means/does, or link to a section in the docs explaining it.

Also:
```suggestion
                    will pick up the run mode from the workflow.
                    
                    .. versionadded:: 8.4.0
```

Reply:
Both - I haven't even started on the accompanying docs PR, but I'm more than happy to improve the detaial here.